src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,28 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+type Props = {
+  post: Post,
+};
+export const PostInfo: React.FC<Props> = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+      {' Posted by  '}
+      <UserInfo user={post.user} />
```

Comment:
```suggestion
      post.user && <UserInfo user={post.user} />
```

Reply:
in this case you don't need check in UserInfo

Reply:
Fix it